<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>image_scan_widget_ui</class>
 <widget class="QWidget" name="image_scan_widget_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>507</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>623</width>
    <height>418</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Scan Widget</string>
  </property>
  <widget class="QFrame" name="image_scan_frame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>450</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="confocal_controls_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QDoubleSpinBox" name="z_confocal_spin_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_confocal_spin_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.005000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y_confocal_spin_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.005000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X(μm)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="show_cursor_radio_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Cursor</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="local_image_scan_push_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Local Scan</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="image_scan_resolution_spin_box">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dwell_time_image_scan_spin_box">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>0.000200000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dwell Time (ms)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y(μm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Z(μm)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="full_image_scan_push_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_cursor_location_push_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Cursor</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="toolbar_frame_image_scan">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
